src/infra/helper/dummyValueGenerator.go

```diff
@@ -0,0 +1,48 @@
+package infraHelper
+
+import (
+	"math/rand"
+)
+
+type DummyValueGenerator struct {
+	generatedUsername string
+}
+
+func (helper *DummyValueGenerator) GenPass(length int) string {
+	charset := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+"
+	charsetLen := len(charset)
+
+	pass := make([]byte, length)
+	for i := 0; i < length; i++ {
+		randomIndex := rand.Intn(charsetLen)
+		pass[i] = charset[randomIndex]
+	}
+
+	return string(pass)
+}
+
+func (helper *DummyValueGenerator) GenUsername() string {
+	dummyUsernames := []string{
+		"yoda", "obi_wan", "anakin", "luke", "leia", "rey", "kylo",
+	}
+	helper.generatedUsername = dummyUsernames[rand.Intn(len(dummyUsernames))]
+
+	return helper.generatedUsername
+}
+
+func (helper *DummyValueGenerator) GenMailAddress(mailUsername *string) string {
+	dummyMailAddressDomains := []string{
+		"@enterprise.com", "@corporation.com", "@company.org",
```

Comment:
If you're using SW chars, then this should be @republic.gov or @senate.gov just saying.